internal/minioclient/client.go

```diff
@@ -117,7 +129,9 @@ func (client *Minio) UploadMultipart(bucket, objectName string, data []byte, par
 }
 
 // Download downloads object data.
-func (client *Minio) Download(bucket, objectName string, buffer []byte) ([]byte, error) {
+func (client *Minio) Download(bucket, objectName string, buffer []byte) (buf []byte, err error) {
```

Comment:
this can be set as `_` instead of `buf` since we don't use that variable, but we do use `err`.

Reply:
done